docs/_components/spinner.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Spinner
+lead: >
+  Spinners are used to help convey that the user should wait for the system to resolve.
+---
+
+Spinners can be helpful to signal to a user that the current process may require a few seconds.
+
+**Before using a spinner** Consider if there is  a way to improve the performance of the sign to mitigate the use of a spinner.
+
+## Example of a spinner
+
+{% capture example %}
+<div class="lg-spinner">
+    <div>
```

Comment:
What do you think about making these code samples 2 space indent instead of 4? It's what we go with for the Sass code and makes it a bit easier to read inside the code sample blocks